### DISCUSSION TITLE
[Question] Consul Leader

### DISCUSSION THREAD

**Discussion by McDefault** [tiangolo/medium-posts#10] [Questions] [answered]
Hello,

About post [docker-swarm-mode-and-distributed-traefik-proxy-with-https/](https://github.com/tiangolo/blog-posts/tree/master/docker-swarm-mode-and-distributed-traefik-proxy-with-https)

Thank you for this post. I have a question about the Consul HA.

Since the leader is predefined in the config, what happens when the Leader crashes/its Node stops working?

**Comment by tiangolo** [comment, marked as answer]
Thanks for the patience!

Sorry, I haven't used that in a very long while, so I wouldn't be in a good position to answer about it anymore. :sweat_smile:
